app/helpers/people_helper.rb

```diff
@@ -1,2 +1,3 @@
 module PeopleHelper
+
```

Comment:
I think these are automatically added by Rails if you use a scaffolding or controller generator. I think that was there before I started to work on LIMS.